README.md

```diff
@@ -157,9 +157,12 @@ point), skipping through all build failures. Once a build finally succeeds, it
 will prompt me to manually check for a segfault and use my input to decide
 whether the current revision is good or bad.
 
-Keep in mind however that this is very early stages. Barely anything is
-documented. I built this to scratch my own itch, and I continue developing it
-whenever I need some feature.
+## Installation
+
+Keep in mind that this tool is in a very early stage of development.
```

Comment:
Please keep the hard-wrapping at 80 characters that is used in the rest of the document. I agree that wrapping at sentence boundaries is much nicer semantically (and I do the same in formats like latex), but one goal of markdown is to be readable in source form. In any case, the style should be consistent.

Reply:
> Please keep the hard-wrapping at 80 characters that is used in the rest of the document. [...] One goal of markdown is to be readable in source form

I feel like semantically line-wrapped prose, while certainly unusual, is perfectly readable in source, but maybe that's just me. The main reason I prefer semantic linebreaks also in MarkDown etc. is that it makes easier to see what changed in subsequend diffs, while necessary re-wrapping to keep a consistent line-lenght often hides the actual changes to the content. (Especially when the diff is viewed in source form rather than the "rich diff" GitHub provides for MarkDown files.)

> In any case, the style should be consistent.

Agreed. Thus re-wrapped to 80 characters.

Reply:
> is perfectly readable in source

Readable yes, but not quite as nice. In markdown the source file is a first-class citizen. Not wrapping at all would also be an alternative, letting the editor take care of it. That would only make the diffs worse though.

> makes easier to see what changed in subsequend diffs

Yes, and it also makes editing easier in many editors (e.g. vim). Very easy to swap around sentence order, delete sentences etc. I like semantic line breaks, but in this case I decided to go for better readability instead. Not 100% sure this is the correct choice, but consistency is king either way ;)